### DISCUSSION TITLE
ReSpeaker 4-Mic Linear Array Kit | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#626] [Q&A]
# ReSpeaker 4-Mic Linear Array Kit | Seeed Studio Wiki

ReSpeaker 4-Mic Linear Array Kit

https://wiki.seeedstudio.com/ReSpeaker_4-Mic_Linear_Array_Kit_for_Raspberry_Pi/

**Comment by manhhungictuk11** [comment]
Hi Jianjing Huang, does this KIT have features such as: beam forming, noise reduction and echo cancellation?
Hope to see your reply. Thanks.

**Reply by Nostalgist7** [reply]
respeaker这个系列产品的软件生命周期已经结束了，也就是说商家不再提供软件更新和维护了。目前这套开发工具只有录音和播放功能，其他的软件算法都要自己开发。

**Reply by Nostalgist7** [reply]
Due to the product software service life cycle has expired, Seeed Studio has ceased updating and maintaining software algorithms for the reSpeaker family of products. This series of products is only used as recording playback module, all algorithm functions need to be developed by yourself.

from:  https://www.seeedstudio.com/ReSpeaker-4-Mic-Linear-Array-Kit-for-Raspberry-Pi.html?queryID=637e069c20a968c61901daf7841ae96f&objectID=132&indexName=bazaar_retailer_products
